tools/ensure_buildomat_artifact.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+#
+# Ensure a buildomat artifact is downloaded and available locally.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Published buildomat artifacts are available at a predictable URL:
+# https://buildomat.eng.oxide.computer/public/file/ORG/REPO/SERIES/HASH/ARTIFACT
+BUILDOMAT_BASE_URL="https://buildomat.eng.oxide.computer/public/file"
+
+# Default value for optional flags
+ORG="oxidecomputer"
+SERIES="image"
+CHECK_HASH=true
+OUTDIR="$PWD"
+
+function usage() {
+    cat <<EOF
+Usage: $0 [OPTIONS] <ARTIFACT> <REPO> <COMMIT>
+
+    REPO:     The repository that published the artifact.
+    ARTIFACT: The name of the artifact.
+    COMMIT:   The commit the artifact was published from.
+
+Options:
+    -o <ORG>    Org containing the repository. [default: $ORG]
+    -s <SERIES> The series artifact was published to. [default: $SERIES]
+    -n          Disable artifact validation.
```

Comment:
small nit: I would use `-f` for "force" instead of `-n`.  `-n` is typically used for "dry run", where a tool
would tel you what it would do, but stop short of actually doing it.

Reply:
a dry run flag might also be nice to add later